core/src/main/java/org/apache/struts2/views/jsp/SetTag.java

```diff
@@ -59,4 +61,32 @@ public void setScope(String scope) {
     public void setValue(String value) {
         this.value = value;
     }
+
+    public void setDoNotTrimBody(String doNotTrimBody) {
```

Comment:
Same here: use boolean.

Reply:
This has been addressed in the most recent commit (the setters and tag annotation now use boolean, as requested).